script.js

```diff
@@ -1,4 +1,489 @@
+  },
+  {
+    name: "10",
+    suit: "♤",
+    rank: 10,
+  },
+  {
+    name: "Jack",
+    suit: "♤",
+    rank: 10,
+  },
+  {
+    name: "Queen",
+    suit: "♤",
+    rank: 10,
+  },
+  {
+    name: "King",
+    suit: "♤",
+    rank: 10,
+  },
+];
+invalid = `<b>You have enter an invalid input!<br>Please only type 1 or 11!`;
+gameMode = "welcome";
+playerName = "";
+playerBets = "";
+blackjack = 21;
+playingChips = 100;
+p1winCounter = 0;
+pcWinCounter = 0;
+drawCounter = 0;
+p1StartHand = [];
+pcHand = [];
+HOSinvalid = `<b>You have entered an invalid entry ❌!<br>Please only enter hit or stand!<br>Your current hand is ${p1StartHand}<br><br>Hit to have additional cards added to your current hand.<br>Stand if you've burst or<br>Stand to finalize your hand!`;
+blackjackMessage = `<b>BLACKJACK!!<br><br>Your final hand is 21!<br>Computer's turn now.`;
+var shuffle = function () {
+  var shuffledeck = deck[Math.floor(Math.random() * 52)];
+  return shuffledeck;
+};
+
+var randomCard = function () {
+  firstDraw = shuffle();
+  secondDraw = shuffle();
+  pDraw = shuffle();
+  cDraw = shuffle();
+  additionalDraw = shuffle();
+  initialHand = [firstDraw.rank + secondDraw.rank];
+  pciHand = [pDraw.rank + cDraw.rank];
+  if (firstDraw.name == "ace" && secondDraw.name == "ace") {
+    gameMode = "pc";
+    p1StartHand = initialHand;
+    p1StartHand = 21;
+    return blackjackMessage;
+  }
+  if (firstDraw.name == "ace" && secondDraw.rank == 10) {
+    gameMode = "pc";
+    p1StartHand = initialHand;
+    p1StartHand = 21;
+    return blackjackMessage;
+  }
+  if (secondDraw.name == "ace" && firstDraw.rank == 10) {
+    gameMode = "pc";
+    p1StartHand = initialHand;
+    p1StartHand = 21;
+    return blackjackMessage;
+  }
+  if (firstDraw.name == "ace" && gameMode == "p1") {
+    gameMode = "chooseAce";
+    p1StartHand = initialHand;
+    return `<b>You drew ${firstDraw.name} of ${firstDraw.suit} and ${secondDraw.name} of ${secondDraw.suit}<br><br>Choose 1 or 11 to represent your ACE.`;
+  }
+  if (secondDraw.name == "ace" && gameMode == "p1") {
+    gameMode = "chooseAce";
+    p1StartHand = initialHand;
+    return `<b>You drew ${firstDraw.name} of ${firstDraw.suit} and ${secondDraw.name} of ${secondDraw.suit}<br><br>Choose 1 or 11 to represent your ACE.`;
+  }
+  if (gameMode == "p1") {
+    p1StartHand = initialHand;
+    gameMode = "p1HOS";
+    return `<b>You drew ${firstDraw.name} of ${firstDraw.suit} and ${secondDraw.name} of ${secondDraw.suit}<br><br>Your starting hand is ${p1StartHand}!`;
+  }
+  if (gameMode == "pc") {
+    gameMode = "gameboard";
+    while (pciHand <= 17) {
+      pciHand = Number(String(pciHand)) + additionalDraw.rank;
+    }
+    return `<b>PC's final hand is ${pciHand}!`;
+  }
+};
+
+var aceOnHand = function (input) {
+  if (firstDraw.name == "ace") {
+    if (input == 1) {
+    }
+    if (input == 11) {
+      p1StartHand = Number(String(p1StartHand)) + 10;
+    }
+  }
+  if (secondDraw.name == "ace") {
```

Comment:
this method is quite hardcoded. 
can it be done through the use of a player card array and the player score so far?